File: lua/refactoring/tests/refactor/106/cpp/prompt_func_param/extract.expected.cpp
#include <stdio.h>

void foo(int a) {
  int test_other = 1;

  for (int i = 0; i < test_other; i++) {
    printf("%d %d", a, test_other);
  }
}


int simple_function(int a) {
  int test = 1;
  foo(a);

  return test;
}

File: lua/refactoring/tests/refactor/106/go/prompt_func_multi_return/extract.expected.go
package main

import "fmt"

func foo(a int) (int, int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
    return test, test_other
}

func simple_function(a int) (int, int) {
    test, test_other := foo(a)


    return test, test_other
}
